index.html

```diff
@@ -207,6 +207,71 @@ <h2 class="headings">Tonics</h2>
 </div>
 
  <div class="card-works">
```

Comment:
- [ ] You did great implementing the WORK section :+1: :+1:  but I think there are some improvements you need to make here before moving to the next milestone. 
- Following the best practices, your entire work section should be wrapped in a `section` tag. this is for semantics purposes as it helps in improving your website SEO. :+1: :smile: 

- I think your project card is missing some items. Looking at the screenshot of the Figma design below, Your design is missing those DOT I circled, As part of this project requirement is to replicate the given design, I'll suggest you go through your design and update these. :+1: :+1: 
- The font color of those language lists ie `html CSS javascript` are not the same as the color from the design, I'll suggest you get the appropriate color and update yours. :+1: :smile: 
- Also I think your button needs a little bit of styling to match the figme design. and currently your button hover sate doesn't match the same background color and text color from the design, I'll suggest you get the correct color and update your CSS. :+1: :smile: :small_airplane: 

FIGMA DESIGN:
![Selection_033](https://user-images.githubusercontent.com/107632780/236300757-f0dd856b-dc1a-4ada-8ef4-d8a052a50d55.png)

YOUR DESIGN:
![Selection_034](https://user-images.githubusercontent.com/107632780/236300826-86abaf4d-9af5-4e8b-9a1e-6dedbf36565f.png)